Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -356,12 +357,18 @@ open class AVFoundationPlayback: Playback {
         do {
             try AVAudioSession.sharedInstance().setCategory(category, mode: .default, options: [.allowAirPlay])
         } catch {
-            print("It was not possible to set the audio session category")
+            Logger.logError("It was not possible to set the audio session category")
         }
     }
-
-    private func handlePlayerRateChanged(player: AVPlayer, changes: Any) {
-        if player.rate == 0 && playerStatus != .unknown && state != .idle {
+    
+    private func handlePlayerRateChanged(player: AVPlayer, changes: NSKeyValueObservedChange<Float>) {
+        guard playerStatus != .unknown else { return }
+        if changes.isPrior && player.rate == 1 && state == .playing {
```

Comment:
O teste nesse condicional não seria `player.rate > 0`?
Teoricamente qualquer valor maior que zero indica que o state do player é `playing`.

Reply:
For some reason I don't remember anymore, we made it playing only when `rate == 1`. I've move it to `rate != 0` again and in other place too. Nothing breaks, so I guess we are good to go :)

Reply:
Great! GTG to me as well :)